lib/tools/adb-commands.js

```diff
@@ -958,18 +963,36 @@ methods.isDataOn = async function isDataOn () {
 };
 
 /**
- * Check the state of animation on the device under test.
+ * Check the state of animation on the device under test below:
+ *   - animator_duration_scale
+ *   - transition_animation_scale
+ *   - window_animation_scale
  *
  * @this {import('../adb.js').ADB}
  * @return {Promise<boolean>} True if at least one of animation scale settings
  *                   is not equal to '0.0'.
  */
 methods.isAnimationOn = async function isAnimationOn () {
-  let animator_duration_scale = await this.getSetting('global', 'animator_duration_scale');
-  let transition_animation_scale = await this.getSetting('global', 'transition_animation_scale');
-  let window_animation_scale = await this.getSetting('global', 'window_animation_scale');
-  return _.some([animator_duration_scale, transition_animation_scale, window_animation_scale],
-                (setting) => setting !== '0.0');
+  return (await B.all(ANIMATION_SCALE_KEYS.map(
+    async (k) => (await this.getSetting('global', k)) !== '0.0'))
+  ).includes(true);
+};
+
+/**
+ * Set animation scale with the given velue via adb shell settings command.
+ *   - animator_duration_scale
+ *   - transition_animation_scale
+ *   - window_animation_scale
+ *
+ * @this {import('../adb.js').ADB}
+ * @param {number} value Animation scale value (int or float) to set.
+ *                       The minimum value of zero disables animations.
+ *                       By increasing the value animations become (slower/faster).
```

Comment:
sorry, I've put `(slower/faster)` into parentheses because it is not clear to me whether the animation is actually getting slower or faster. Do you know what to put there?

Reply:
ah, gotcha. Increasing -> then the animation is being slower. 0 is the fastest (no animation). Let me modify that